frontend/app/utils/api.tsx

```diff
@@ -66,13 +66,24 @@ export const suggestExtractor = async ({
   return response.data;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const runExtraction: MutationFunction<any, any> = async (
-  extractionRequest,
-) => {
+type ExtractionRequest = {
+  extractor_id: string;
+  text?: string;
+  file?: File;
+};
+
+type ExtractionResponse = {
+  data: unknown[];
+};
```

Comment:
Is this data structure actually unknown? If possible we should type it

Reply:
Also build failing because `unknown` is not assignable to `Record<string, any>[]`.

Three options:
1. switch type to `Record<string, any>[]` if we know the response will always be a dict, then this is valid
2. switch to `any[]`
3. add typing for the response object

Reply:
Yep I'll adjust in a bit